NSS-Spontaneous-Spoonbills/NSS-Spontaneous-Spoonbills
# baseline file structure

## Link to Ticket
No ticket
Autolink format: hashtag and number, ex: #1094

Closes #

## Description of Proposed Changes
Created folder called 'bangazonapi' that will store models/views/etc for entire project
Started on faker_factory.
Created all files in models folder

## Steps to Test
```
git fetch --all
git checkout patrick
```
Outline the steps to test
Check API is sibling of bangazon folder which is our admin folder
Check models folder is inside bangazonapi
disregard models.py.  this will be deleted.  all models will be created within model directory
still working on how to access foreign keys in models
```
git fetch --all
git checkout <feature_branch>
```

## Impacted Areas in Application

List general components of the application that this PR will affect:

## Mentions @username

Tag users that need to review this code

## Testing

[ ] There are new unit tests in this PR, and I verify that there is full coverage of all new code.
[ ] I certify that all existing tests pass

## Deploy Notes

Notes regarding deployment the contained body of work.  These should note any
db migrations, etc.

## Definition of Done

1. The project must be fully documented. This includes the following:
   1. Complete README that documents the steps to install the code, how to install any dependencies, or system configuration needed.
   1. Every class must be documented with purpose, author, and methods.
   1. Every method must be documented with purpose and argument list - which itself must contain a short purpose for each argument.
1. The project must be able to run fully, and without errors, on each teammate's system.
1. Fulfills every requirement.
1. Every line of code has been peer reviewed.
1. For projects that require unit testing, core functionality must be identified and have at least one test for each
